src/generators/AbstractGenerator.ts

```diff
@@ -34,28 +34,58 @@ export abstract class AbstractGenerator<Options extends CommonGeneratorOptions =
   }
 
   public abstract render(model: CommonModel, inputModel: CommonInputModel): Promise<RenderOutput>;
+  public abstract renderFull(model: CommonModel, inputModel: CommonInputModel, options: RenderFullOptions): Promise<RenderOutput>;
```

Comment:
```suggestion
  public abstract renderFileContent(model: CommonModel, inputModel: CommonInputModel, options: RenderFullOptions): Promise<RenderOutput>;
```

It should say more that `renderFull`.

Reply:
see https://github.com/asyncapi/modelina/pull/401#discussion_r723278001